setup.py

```diff
@@ -29,7 +29,7 @@
     extras_require={"dev": dev_requires},
     entry_points={
         "prefect.collections": [
-            "SnowflakeCredentials = prefect_snowflake.credentials",
+            "prefect_snowflake = prefect_snowflake",
```

Comment:
Not necessarily related to the scope of this PR, but we need an import of `SnowflakeConnector` in `prefect_snowflake/__init__.py` also to make sure it's loadable. Can you either add that to this PR or create an issue for it to make sure it gets done?